nucliadb/requirements.txt

```diff
@@ -1,2 +1,4 @@
 uvicorn
-pydantic_argparse
\ No newline at end of file
+pydantic_argparse
+nucliadb-one>=1.0.0
```

Comment:
nucliadb has dependency with nucliadb_writer. Now is solved with nucliadb_one, but we want to be more explicit?